modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/configuration/CETConfigurationFactory.java

```diff
@@ -74,25 +108,112 @@ protected void modified(Map<String, Object> properties) throws Exception {
 				_cet = _cetManager.addCET(
 					cetConfiguration, companyId,
 					_getExternalReferenceCode(properties));
+
+				if (Objects.equals(
+						_cet.getType(),
+						ClientExtensionEntryConstants.TYPE_THEME_CSS)) {
+
+					_clientExtensionEntryRelLocalService.
+						deleteClientExtensionEntryRels(
+							companyId, _cet.getExternalReferenceCode());
+
+					_addControlPanelThemeCSSClientExtensionEntryRel(companyId);
+				}
 			});
 	}
 
+	private void _addControlPanelThemeCSSClientExtensionEntryRel(Long companyId)
+		throws PortalException {
+
+		ThemeCSSCET themeCSSCET = (ThemeCSSCET)_cet;
+
+		if (!Objects.equals(themeCSSCET.getScope(), "controlPanel")) {
+			return;
+		}
+
+		ClientExtensionEntryRel clientExtensionEntryRel =
```

Comment:
what about calling this existingControlPanelThemeCSSClientExtensionEntryRel or currentControlPanelThemeCSSClientExtentensionEntryRel?

Or if that's too long just existingClientExtensionEntryRel or currentClientExtentensionEntryRel?

Reply:
Not necessary, it's the only `ClientExtensionEntryRel` here

Reply:
Okay.